src/windowsdesktop/src/windowsdesktop/src/bundle/Wix.targets

```diff
@@ -262,8 +191,8 @@
       <DefineConstants>$(DefineConstants);MajorVersion=$(MajorVersion)</DefineConstants>
       <DefineConstants>$(DefineConstants);MinorVersion=$(MinorVersion)</DefineConstants>
       <DefineConstants>$(DefineConstants);WinFormsAndWpfVersion=$(WinFormsAndWpfVersion)</DefineConstants>
-      <DefineConstants>$(DefineConstants);DotNetRuntimeVersion=$(DotNetRuntimeVersion)</DefineConstants>
-      <DefineConstants>$(DefineConstants);AspNetCoreVersion=$(AspNetCoreVersion)</DefineConstants>
+      <DefineConstants>$(DefineConstants);DotNetRuntimeVersion=$(MicrosoftNETCoreAppRefVersion)</DefineConstants>
+      <DefineConstants>$(DefineConstants);AspNetCoreVersion=$(MicrosoftNETCoreAppRefVersion)</DefineConstants>
```

Comment:
This value is also unused. And also I don't think aspnetcore is actually necessarily even built when the WD installers are getting built:

```
PS C:\d\src\windowsdesktop> git grep -i AspNetCoreVersion
src/windowsdesktop/src/bundle/Wix.targets:      <DefineConstants>$(DefineConstants);AspNetCoreVersion=$(MicrosoftNETCoreAppRefVersion)</DefineConstants>
src/windowsdesktop/src/bundle/bundle.wxs:    <Variable Name="ASPNETCOREVERSION" Type="string" Value="$(AspNetCoreVersion)" bal:Overridable="no" />
```